tests/foreman/ui/test_org.py

```diff
@@ -75,24 +104,20 @@ def test_positive_create_with_parent(self, parent_name):
         """
         org_name = gen_string('alpha')
         with Session(self.browser) as session:
-            make_org(session, org_name=parent_name)
-            make_org(
-                session,
-                org_name=org_name,
-                label=gen_string('alpha'),
-                desc=gen_string('alpha'),
-                parent_org=parent_name
-            )
-            self.assertIsNotNone(self.org.search(org_name))
+            for parent_name in generate_strings_list():
```

Comment:
I'll have to disagree :) We don't want to commit code that is incorrect and would amount to code debt for us. The fix is fairly simple and I think worth the time to include it here. The 'old' code did not have this issue due to the nature of DDT-based tests and if we merge this as it, it would amount to a regression. Mind fixing it, please?